tests/Fhi.HelseIdSelvbetjening.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -102,6 +101,60 @@ public async Task UpdateClientKeysFromParameters()
             await host.StopAsync(TimeSpan.FromSeconds(10));
         }
 
+        [Test]
+        public async Task VerifyNewKeys_WithGeneratedKey_Succeeds()
+        {
+            var originalOutput = Console.Out;
+            var originalError = Console.Error;
+
+            await using var stringWriter = new StringWriter();
+            await using var errorWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+            Console.SetError(errorWriter);
+
+            try
```

Comment:
using try/catch to encapsulate the full body of a test block, will swallow the Assert failures, since these are just exception triggers when failing.